docs/multicluster/user-guide.md

```diff
@@ -559,6 +560,42 @@ In future releases, some additional tooling may become available to automate the
 creation of ResourceExports for ACNPs, and provide a user-friendly way to define
 Multi-cluster NetworkPolicies to be enforced in the ClusterSet.
 
+## Multicluster Pod to Pod Connectivity
+Since Antrea v1.9.0, Multi-cluster supports routing Pod traffic across clusters through the Multi-cluster Gateways.
```

Comment:
Please wrap lines at column 80 as other lines in the doc.

Reply:
Done. As well as another section.